File: python/nivel 2 logica/09.py
"""
Ejercicio 9
Multiplicar dos números sin usar el operador *
"""
num1 = int(input("Ingrese el primer numero: "))
num2 = int(input("Ingrese el segundo numero: "))

resultado = 0

for _ in range(num2):
  resultado += num1

print(f"el resultado de la multiplicacion es: {resultado}")

File: javascript/nivel 2 logica/09.js
/*
Ejercicio 9
Multiplicar dos números sin usar el operador *
*/

let num1 = parseInt(prompt("Ingrese el numero 1: "));
let num2 = parseInt(prompt("Ingrese el numero 2: "));

let resultado = 0;

for (let i = 0; i < num2; i++) {
  resultado += num1;
}

console.log("El resultado de la multiplicacion es: ", resultado);

